### PR TITLE
added kadane's algo and dijkstra's algo

### PR DESCRIPTION
**Pull request by shloksuman**
I have added kadane's algo and dijkstra's algo in c++

### CODE REVIEW

File: Algorithms/dijkstras/dijkstrasalgorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Contributed by - Shlok Kumar Suman shloksuman01@gmail.com
```
Comment (harshit-sharma-gits) [inline Algorithms/dijkstras/dijkstrasalgorithm.cpp:1]: enclose your email in angle brackets

File: Algorithms/kadanesalgo/kadanesalgo.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// Contributed by - Shlok Kumar Suman shloksuman01@gmail.com
```
Comment (harshit-sharma-gits) [inline Algorithms/kadanesalgo/kadanesalgo.cpp:1]: here too, enclose your email in angle brackets

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
Can you rename the folder `kadanesalgo` to `kadane`, as it will suffice.

Also, please rename your files as well:
* `dijkstrasalgorithm.cpp` -> `dijkstras.cpp`
* `kadanesalgo.cpp` -> `kadane.cpp`

**Comment by shloksuman** [comment]
updated

**Review by harshit-sharma-gits — APPROVED** [review]
Awesome work @shloksuman. Approving!

**Comment by shloksuman** [comment]
> Awesome work @shloksuman. Approving!

Thank you 😇 @harshit-sharma-gits

**Pull request merged**